bc_obps/reporting/models/electricity_import_data.py

```diff
@@ -0,0 +1,101 @@
+from django.db import models
+from registration.models.time_stamped_model import TimeStampedModel
+from reporting.models.report_version import ReportVersion
+from reporting.models.triggers import immutable_report_version_trigger
+from reporting.models.rls_configs.electricity_import_data import Rls as ElectricityImportDataRls
+
+
+class ElectricityImportData(TimeStampedModel):
```

Comment:
This breaks the convention of naming all our report data `ReportSomethingSomething`, I think we should name it `ReportElectricityImport`